recipe/meta.yaml

```diff
@@ -28,7 +28,9 @@ requirements:
     - bleach >=2.1.3,<3
     - boost-cpp >=1.61.0,<2
     - brial >=1.0.1,<2
-    - cddlib >=0.94g
+    # SageMath uses 0.94g but that does not seem to be packaged in conda-forge,
+    # 0.94j does not have cdd_both_reps anymore
+    - cddlib >=0.94g,<0.94j
```

Comment:
Can you fix this to be `>=1!0.94g,<1!0.94j`? Then we can merge the cddlib upgrade PR.

Reply:
Ok. I had thought that "1!" was the implicit epoch.